packages/react-heartwood-components/src/components/Message/Message.js

```diff
@@ -81,53 +98,75 @@ export const Message = (props: MessageProps) => {
 		}
 
 		return (
-			<p class="message__reply">
+			<p className="message__reply">
 				{icon && (
 					<Icon icon={icon} className={cx('message__reply-icon', iconClass)} />
 				)}
-				<span class="message__reply-text">{children}</span>
+				<span className="message__reply-text">{children}</span>
 			</p>
 		)
 	}
 
 	const renderAttachment = attachment => {
-		return <div class="message__attachment">{attachment}</div>
+		return <div className="message__attachment">{attachment}</div>
 	}
 
-	return (
-		<div className={cx('message', className)}>
-			<span class="message__pre">
+	const renderImage = ({ image, alt, isFromSprucebot }) => {
+		if (isFromSprucebot) {
+			return (
 				<img
-					class="message__from-image"
-					src={fromImage || ''}
-					alt={fromAlt || fromName || ''}
+					className="message__from-image message__from-sprucebot-image"
+					src={image || ''}
```

Comment:
Probably return `null` if you don't have a source, this would just render a busted image